src/components/Main/Schedule/Table.tsx

```diff
@@ -24,23 +25,63 @@ import { makeStyles, Theme, createStyles } from '@material-ui/core/styles';
 import { fade, darken } from '@material-ui/core/styles';
 
 export const Table = ({ columns, data, updateMyData, title, access }: TableProps) => {
+  const outerRef = useRef(null);
   const classes = useStyles();
+  const [dataRows, setDataRows] = useState(data);
+  useEffect(() => {
+    setDataRows(data);
+  });
```

Comment:
does it make sense to add the array param? maybe it'll fix the double-render
```suggestion
  }, [data]);
```
or do you even need `useEffect` if `dataRows` already defaults to `data` when you use `useState(data)`?

Reply:
hm it doesn't seem to work without this useeffect, i'm not really sure why. I'll add the array param